### ISSUE TITLE
Unable to use GraphViz

### ISSUE THREAD

**Issue by robross0606** [anb0s/eclox#163]
GraphViz (dot) worked okay with eclox 0.8.  However, ever since I updated to Eclox 0.11, the same doxyfile and source is generating nothing but errors:

```
error: Problems running dot: exit code=-1, command='dot', arguments='"C:/Code/ATM/ATM/docs/html/d5/d13/a00205_a6c6592e035469f44bc078036077a06ef_icgraph.dot" -Tpng -o "C:/Code/ATM/ATM/docs/html/d5/d13/a00205_a6c6592e035469f44bc078036077a06ef_icgraph.png" -Tcmapx -o "C:/Code/ATM/ATM/docs/html/d5/d13/a00205_a6c6592e035469f44bc078036077a06ef_icgraph.map"'
```

And, yes, Graphviz is correctly installed and on the path:

```
dot - graphviz version 2.38.0 (20140413.2041)
```

Also, when I run doxygen with this same doxyfile outside of eclox, dot is used correctly and everything works fine.  The issue only occurs with Eclox.

**Comment by anb0s** [comment]
Thanks for reporting! I've pushed the testing (for internal use only) version 0.11.x to the new update site by mistake :( sorry, now i've deleted it from update site now...
You can remove it and install the 0.8.x version again.

Or if you still want playing with new version, go on, i will put some updates until release...

**Comment by anb0s** [comment]
The new version from GitHub has the issue, investigating...

**Comment by anb0s** [comment]
fixed for 0.11.0

Now the default and also other Eclipse variables are added to the environment.

If you want to verify before release: published testing build to: http://anb0s.github.io/eclox/testing

Thanks!

**Comment by anb0s** [comment]
similar problem reported here:
https://e2e.ti.com/support/development_tools/code_composer_studio/f/81/t/550404
